contributors/devel/development.md

```diff
@@ -279,20 +279,38 @@ To build binaries for all platforms:
 make cross
 ```
 
+#### Verification
+```
+# Run all hack/verify-*.sh
+make verify
+```
+
 #### Test
 
 ```sh
 cd $working_dir/kubernetes
 
+# Run all hack/update-*.sh
+# For small change, run "make verify" and then relevant scripts, for example:
+# hack/update-gofmt.sh (when adding new files)
+# hack/update-bazel.sh (when updating dependencies / imports)
+make update
+
 # Run every unit test
 make test
 
 # Run package tests verbosely
 make test WHAT=pkg/util/cache GOFLAGS=-v
+# Or
+bazel test //pkg/util/cache/...
 
 # Run integration tests, requires etcd
+# Tip: if you don't have it already installed, run "hack/install-etcd.sh" beforehand
 make test-integration
 
+# Build e2e test binaries
+make all WHAT=test/e2e/e2e.test
```

Comment:
I wonder if the Makefile should have an easier way to build the tests without running them. Thoughts?